src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -737,6 +737,40 @@ void M()
             }
         }
 
+        [Fact]
+        public void StatementAttributeSemanticModel()
```

Comment:
I just wasn't sure where to put this one test, there are no statement attribute binding tests or semantic model tests that I could find.